about.html

```diff
@@ -35,36 +35,46 @@ <h3>About us</h3>
             <aside>
                 <p>CHIP WISSEN | Germany</p>
             </aside>
-            <h3>Faszination Wissenschaft</h3>
-            <p>Jetzt wird's spannend – mit Mathematik, Physik, Chemie und Biologie! Falls diese<br>Themen bisher nicht
-                zu Ihren bevorzugten Interessensgebieten gezählt haben,<br>geht es Ihnen nicht anders als der Mehrzahl
-                der deutschen Schüler, wie der<br>letzte Pisa-Test 2015 ergeben hat. Denn demnach<br>war das Interesse
-                an naturwissenschaftlichen Fächern trotz insgesamt guter<br>Leistungen eher unterdurchschnittlich. Wie
-                schade, offensichtlich gelingt es den<br>Schulen nur unzureichend, Motivation für die Beschäftigung mit
-                den<br>sogenannten MINT-Fächern zu wecken.</p>
+            <h3><span lang="de">Faszination Wissenschaft</span></h3>
+            <p><span lang="de">Jetzt wird's spannend – mit Mathematik, Physik, Chemie und Biologie! Falls
```

Comment:
Span element improved language recognition and solved previous issues related to paragraph content indentation. Good job